docs/source/getting-started-kernels.md

```diff
@@ -7,12 +7,22 @@ The following kernels have been tested with the Jupyter Enterprise Gateway:
 * Scala 2.11/Apache Spark 2.x with Apache Toree kernel
 * R/Apache Spark 2.x with IRkernel
 
-We provide sample kernel configuration and launcher tar files as part of [each release](https://github.com/jupyter/enterprise_gateway/releases) (e.g. [jupyter_enterprise_gateway_kernelspecs-2.0.0.dev2.tar.gz](https://github.com/jupyter/enterprise_gateway/releases/download/v2.0.0.dev2/jupyter_enterprise_gateway_kernelspecs-2.0.0.dev2.tar.gz)) that can be extracted and modified to fit your configuration.
+We provide sample kernel configuration and launcher tar files as part of [each release](https://github.com/jupyter/enterprise_gateway/releases) (e.g. [jupyter_enterprise_gateway_kernelspecs-2.0.0.dev2.tar.gz](https://github.com/jupyter/enterprise_gateway/releases/download/v2.0.0rc1/jupyter_enterprise_gateway_kernelspecs-2.0.0rc1.tar.gz)) that can be extracted and modified to fit your configuration.
 
 Please find below more details on the specific kernels supported by Enterprise Gateway.  In each of the sections we set the `KERNELS_FOLDER` to `/usr/local/share/jupyter/kernels` since that's one of the default locations searched by the Jupyter framework.  Co-locating kernelspecs hierarchies in the same parent folder is recommended, although not required.
 
 For information about how to build your own kernel-based docker image for use by Enterprise Gateway see [Custom kernel images](docker.html#custom-kernel-images).
 
+You will find in this documentation details on how to install the kernels on various orchestrators:
+
++ [Hadoop Yarm client mode](kern-yarn-client-mode.md.html).
++ [Hadoop Yarm cluster mode](kern-yarn-cluster-mode.html).
```

Comment:
Please fix the typos of these Yarn entries and let's drop the period at the end of each of these bullets.

Reply:
fixed